test_util/test_upgrade_vpc.py

```diff
@@ -115,6 +115,9 @@ def main():
     stable_installer_url = os.environ['STABLE_INSTALLER_URL']
     installer_url = os.environ['INSTALLER_URL']
 
+    config_yaml_override_install = os.getenv('CONFIG_YAML_OVERRIDE_INSTALL')
+    config_yaml_override_upgrade = os.getenv('CONFIG_YAML_OVERRIDE_UPGRADE')
```

Comment:
I would suggest removing the "override" term from these variables. We have other environment variables which are also overriding defaults (see beginning of this file), but we don't explicitly call out that fact in the variable name.